src/Migrations/M036RemoveEnFormOptions.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+
+/**
+ * Remove the Theme Editor feature flag from Planet 4 features.
+ */
+class M036RemoveEnFormOptions extends MigrationScript
```

Comment:
Should we also clean up the block code?

![Edit-Page-“All-blocks-Evergreen-template”-‹-Greenpeace-—-WordPress](https://github.com/user-attachments/assets/88056f71-dc26-4998-91aa-7a3b0493f2aa)

Reply:
I was a bit hesitant on touching live content. But since existing forms are already broken, because of EN API sunset, we can remove them completely. I adjusted a bit the migration.